### ISSUE TITLE
atomic and thread

### ISSUE THREAD

**Issue by wizardforcel** [AnthonyCalandra/modern-cpp-features#28]
C++11 also introduced atomic and thread, but cannot be seen here.

**Comment by AnthonyCalandra** [comment]
Lots of stuff is missing -- I wish I had more free time! Thanks for pointing it out, I'll get to some of the concurrency stuff once I have some free time.

**Comment by conz27** [comment]
@AnthonyCalandra ... might consider opening up an issue with "help wanted" label and let others help contribute?

**Comment by AnthonyCalandra** [comment]
@conz27 Perhaps that could work... I've been thinking of how to get more content on here. I thought this would be a lot easier considering anyone can just submit pull requests.

**Comment by conz27** [comment]
If people know you need help with something, they might be more willing to contribute rather than expect you to do it. Just food for thought.

**Comment by stefanos82** [comment]
Another suggestion would be to split the file into different `.md` files, such as `CPP_11.md`, 'CPP_14.md`, and `CPP_17.md`.

This way it would be a lot easier to follow each standard to study it separately without following a top to bottom concept that is a bit overwhelming in my humble opinion.

I find it confusing to start reading about the latest C++17 first and then have to scroll all the way down to the bottom to read about C++11 new features that I am more interested in.

Basically the `README.md` should played the role of the table of contents that lead the user to a different `.md` file that is fully dedicated to each C++ new-features release.

What do you think?

**Comment by AnthonyCalandra** [comment]
Good suggestion! I'll open up a new issue to track this.

**Comment by riktimmondal** [comment]
Please can you tell more briefly what do we need to do actually since i am beginner

**Comment by AnthonyCalandra** [comment]
If you're asking about contributing features to the sheet then I have a guideline you can follow here: https://github.com/AnthonyCalandra/modern-cpp-features/blob/master/CONTRIBUTING.md.

Otherwise, please open a new issue and I can try to help.

**Comment by tupaschoal** [comment]
I've written a section on each, if both PRs get pushed, I think we can close this issue 👍
